src/vue/text-editor/text-editor-single-block.vue

```diff
@@ -78,46 +81,43 @@ export default {
     }
   },
   watch: {
-    text (val) {
-      if ((!this.prevText && val) || (this.prevText && !val)) {
-        this.updateText()
-      }
-      this.prevText = val
-    },
     async '$store.state.optionsUpdated' () {
       if (!this.localTextEditorOptions.ready && this.$settingsC.tokenizerOptionsLoaded) {
         await this.prepareDefaultTextEditorOptions()
       }
     },
-    '$store.state.tokenizerUpdated' () {
-      this.updateText()
+    '$store.state.uploadCheck' () {
+      this.updateFromExternal()
     },
     async '$store.state.alignmentRestarted' () {
       await this.restartTextEditor()
     }
   },
   computed: {
+    formattedTextId () {
+      return this.textId ?? 'no-id'

```

Comment:
is this value relied upon anywhere else? If so, maybe it would be better to make this a static constant on the alignedtext class (or somewhere) ?

Reply:
No, it is not used anywhere else. It is defined only for constructing ids (container, remove icon, textarea).
So I believe it is redundant to define it elsewhere

Reply:
ok